packages/docs/pages/reference/elements/col.mdx

```diff
@@ -8,12 +8,76 @@ import {
 
 # `<col>`
 
-Short description of what the `<col>` tag does.
+The `<col>` tag is optionally used in a [`<tabular>`](tabular) environment.
+ Its primary use is to define horizontal alignment within a column or to 
+ specify by percentage how much of the tabular environment the column should
+  occupy.
+
+If the `<col>` tag does appear, it must occur prior to any rows and must occur
+exactly as many times as there will be columns.
+
 
 ## Syntax
 
 <SyntaxDisplay name="col" />
 
+## Notes
+
+The `width` option will only be applied to columns containing a cell with 
+a `<p>`.  Conversely, the use of a `<p>` tag inside a cell necessitates the
+ use of a `<col>` preamble to the rows of a `<tabular>`.
+
 ## Examples
 
-    
\ No newline at end of file
+```ptx-example
+<FRAGMENT parents="section subsection">
+    <table>
+          <title>Column options (top="minor" right="medium")</title>
```

Comment:
We don't need to say the options `top` and `right` again. They can be clearly seen in the source.